plugin/src/main/java/org/jenkinsci/plugins/casc/CasCGlobalConfig.java

```diff
@@ -0,0 +1,46 @@
+package org.jenkinsci.plugins.casc;
+
+import hudson.Extension;
+import jenkins.model.GlobalConfiguration;
+import net.sf.json.JSONObject;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.StaplerRequest;
+
+/**
+ *
+ */
+@Extension
+public class CasCGlobalConfig extends GlobalConfiguration {
```

Comment:
maybe simpler for you to make the ConfigurationAsCode class extend GlobalConfiguration ?

Reply:
At this point it makes a perfect sense to do this. The question is not what is easy for me, but what is the best approach to have this feature completed.

Reply:
do what is simpler for you, we always can rework later